tests/fe/point_evaluation_01.cc

```diff
@@ -21,7 +21,6 @@
 
 #include <deal.II/dofs/dof_handler.h>
 
-#include <deal.II/fe/fe_point_evaluation.h>
```

Comment:
By the way, do you also want to move the tests into the `matrix_free` folder? As of now, the matrix-free folder is still somewhat smaller, so it is at least no regression.

Reply:
Done!